src/v2/Apps/Artist/Components/ArtistTopWorksRail/ArtistTopWorksRail.tsx

```diff
@@ -0,0 +1,129 @@
+import { Box, Flex, Sans } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtistTopWorksRail_artist } from "v2/__generated__/ArtistTopWorksRail_artist.graphql"
+import { Carousel } from "v2/Components/Carousel"
+import FillwidthItem from "v2/Components/Artwork/FillwidthItem"
+import { Media } from "v2/Utils/Responsive"
+import { StyledLink } from "v2/Apps/Artist/Components/StyledLink"
+import { scrollIntoView } from "v2/Utils/scrollHelpers"
+import { useTracking } from "v2/Artsy"
+import * as Schema from "v2/Artsy/Analytics/Schema"
+import styled from "styled-components"
+
+interface ArtistTopWorksRailProps {
+  artist: ArtistTopWorksRail_artist
+  onOverviewTab?: boolean
+}
+
+/**
+ * The H2 tag was added for SEO purposes
```

Comment:
Ditto here (the comment is more likely to be removed eventually if it's right above the element it's referring to. 😄 )